pipelines/batch/setup_prod_job.py

```diff
@@ -242,5 +305,7 @@ def main(
 if __name__ == "__main__":
     args = parser.parse_args()
     args.diseases = args.diseases.split()
-    args.excluded_locations = args.excluded_locations.split()
+    if args.locations_include is not None:
+        args.locations_include = args.locations_include.split()
+    args.locations_exclude = args.locations_exclude.split()
```

Comment:
Seems like we should either have the `is not None` check for both or neither, unless I am missing something.

Reply:
`--locations-exclude` defaults to a string, not`None`.

Reply:
(and if it is explicitly set to the empty string that will work, as it will result in an empty list)